soci/soci_index.go

```diff
@@ -50,6 +50,12 @@ const (
 	IndexAnnotationBuildToolIdentifier = "com.amazon.soci.build-tool-identifier"
 	// media type for OCI Artifact manifest
 	OCIArtifactManifestMediaType = "application/vnd.oci.artifact.manifest.v1+json"
+	// default span size
```

Comment:
can you please also comment what the value is, e.g. 4MiB?

Reply:
done